k-distribution/include/kframework/builtin/domains.md

```diff
@@ -1055,6 +1059,14 @@ module INT-SYMBOLIC [symbolic]
   rule X modInt N => X requires 0 <=Int X andBool X <Int N [simplification]
   rule X   %Int N => X requires 0 <=Int X andBool X <Int N [simplification]
 
+  rule  1 ^Int _ =>  1                            [simplification]
+  rule -1 ^Int N =>  1 requires N %Int 2 ==Int  0 [simplification]
+  rule -1 ^Int N => -1 requires N %Int 2 =/=Int 0 [simplification]
+  rule  0 ^Int N =>  0 requires N >Int 0          [simplification]
+  rule  X ^Int N =>  0 requires absInt(X) >Int 1
+                        andBool N <Int 0          [simplification]
+  rule  X ^Int 0 =>  1 requires X =/=Int 0        [simplification]
```

Comment:
Have we double-checked that `_%Int_` is behaving correctly on negative numbers here?

I remember that `_%Int_` and `_modInt_` had different behaviors for negative inputs on the first number, I just want to make sure we either get the right one, or write this code so that it doesn't matter which one.